locales/en.json

```diff
@@ -2016,16 +2076,20 @@
         "btn-confirm": "View",
         "btn-reconfirm": "Review",
         "btn-view": "View",
+        "btn-select-account": "Select account",
+        "select-refund-account": "Select refund account",
+        "no-payment-method-on-quote": "No payment method given",
         "status": "Status",
         "refund-status-label": "Refund status",
         "detailed-quote-status": {
           "pending-user-confirmation": "You must review your quote",
           "cancelled-by-user": "You declined this quote",
-          "failed-expired-on-bank-transfer": "Payment not received",
+          "failed-expired-on-bank-transfer": "Payment not recieved",
```

Comment:
This is definitely wrong..not sure how these got through in the other repo

Reply:
@amsul Just for the record -- these are errors in the original file that were corrected by @Marcpepe , but I reversed his original PR because we need to fix them in the private repo, not the public one. 

I have the list of corrections and will deal with them in the proper place next week.